### PR TITLE
change "rsgname" to "groupname" 

### PR DESCRIPTION
**Pull request by SmartKeyerror**
change "rsgname" to "groupname"  on the view of gp_resgroup_status 
and gp_resgroup_status_per_host, make them consistent.

### CODE REVIEW

File: src/test/isolation2/expected/resgroup/resgroup_views.out [hunk 1]
```diff
@@ -34,17 +34,17 @@ select * from gp_toolkit.gp_resgroup_config;
  6441    | system_group  | 0           | 10                   | 100               | -1     | -1           | 500      
 (3 rows)
 select * from gp_toolkit.gp_resgroup_status;
- rsgname       | groupid | num_running | num_queueing | num_queued | num_executed | total_queue_duration | cpu_usage                                     
----------------+---------+-------------+--------------+------------+--------------+----------------------+-----------------------------------------------
- default_group | 6437    | 0           | 0            | 0          | 0            | @ 0                  | {'-1': 0.0, '0': 0.0, '1': 0.0, '2': 0.0}     
- admin_group   | 6438    | 1           | 0            | 0          | 13           | @ 0                  | {'-1': 0.33, '0': 0.12, '1': 0.09, '2': 0.05} 
- system_group  | 6441    | 0           | 0            | 0          | 0            | @ 0                  | {'-1': 0.09, '0': 0.08, '1': 0.08, '2': 0.08} 
+ groupid | groupname     | num_running | num_queueing | num_queued | num_executed | total_queue_duration
```
Comment (kainwen) [inline src/test/isolation2/expected/resgroup/resgroup_views.out:37]: Why missing `cpu_usage` column?

Comment (SmartKeyerror) [reply src/test/isolation2/expected/resgroup/resgroup_views.out:37]: Already removed this field from `gp_resgroup_status`

Comment (kainwen) [reply src/test/isolation2/expected/resgroup/resgroup_views.out:37]: > Already removed this field from `gp_resgroup_status`

Previous commit does not modify this test case?

Comment (SmartKeyerror) [reply src/test/isolation2/expected/resgroup/resgroup_views.out:37]: Yes, the results have been ignored, so the test will not fail even if we removed the cpu_usage field from the view.

**Review by kainwen — APPROVED** [review]

**Comment by SmartKeyerror** [comment]
My local dev pipeline tests are all passed, merge it.

**Pull request merged**